example/application/container.tsx

```diff
@@ -215,6 +215,7 @@ const Viewer: React.FunctionComponent<ViewerProps> = ({ document }): React.React
               source={document.frameSource}
               dispatch={fromFrame}
               onConnected={fn}
+              onConnectionFailure={(setDocumentToRender) => setDocumentToRender(document.document)}
```

Comment:
Can we highlight this part in the README somewhere? To recommend that users set this to enable the default template